Jenkinsfile

```diff
@@ -0,0 +1,2 @@
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
+buildPlugin(configurations: buildPlugin.recommendedConfigurations())
```

Comment:
great, I was unsure this would work also for modules, and hadn't tried yet :+1:

Reply:
It works, but see [JENKINS-55582](https://issues.jenkins-ci.org/browse/JENKINS-55582): I just want to `s/jenkins-module/hpi/` anyway!